src/values/source/sourceTypes/TextSource.ts

```diff
@@ -55,7 +55,7 @@ class TextSource extends BaseSource {
 
     const tmpAction = new Action();
 
-    tmpAction.addQualifier(new Qualifier('l', layerParam));
+    tmpAction.addQualifier(new Qualifier(layerType, layerParam));
```

Comment:
this fixes a lint error `unsed variable`, but also fixes a bug that we forgot to use the layerType.